docs/src/components/pricing/PricingTable.tsx

```diff
@@ -922,11 +931,11 @@ export default function PricingTable({
       <RowHead startIcon={<IconImage name="product-core" width="28" height="28" />}>
         MUI Core (open-source)
       </RowHead>
+      {renderRow('Material UI')}
```

Comment:
So it matches our order in

<img width="550" alt="Screenshot 2022-05-17 at 19 57 23" src="https://user-images.githubusercontent.com/3165635/168879114-3a88b1f0-8f0a-4756-8fa7-9dab449bc9a2.png">